packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-state.ts

```diff
@@ -77,9 +79,24 @@ export function getGradableGraph(
     }
 
     if (state.type === "angle" && initialGraph.type === "angle") {
+        // We're going to reverse the coords for scoring if the angle is clockwise and
+        // we don't allow reflex angles. This is because the angle is largely defined
+        // by the order of the points in the coords array, and we want to maintain
+        // the same angle scoring with the legacy graph (which had a bug).
+        // (LEMS-2190): When we remove the legacy graph, move this logic to the scoring function.
```

Comment:
Thanks for leaving such a great, explanatory comment.  I was thinking about removing the old code and how we'd know about LEMS-2190 at that time.  What do you think of also adding a reference to LEMS-2190 at the top of the legacy interactive-graph file?  Or near some code that we know we'll be touching when we delete it.